### DISCUSSION TITLE
ScriptReference/Physics2D.SyncTransforms

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#634] [Notes]
# ScriptReference/Physics2D.SyncTransforms

https://docs.unity3d.com/ScriptReference/Physics2D.SyncTransforms.html

**Comment by qwe321qwe321qwe321** [comment]
It is used to flush all Transform changes to physics (rigidbodies), not vice versa.
[from the Unity dev.](https://forum.unity.com/threads/physics-synctransforms-questions.487424/#post-3180021)
